pkg/arvo/lib/hood/drum.hoon

```diff
@@ -663,6 +664,8 @@
     ^+  +>
     (ta-poke %sole-action !>(act))
   ::
+  ++  ta-duct  (cat 3 'drum_' (scot %p our.hid))      ::  per-ship duct
```

Comment:
This isn't technically a "duct", though it's similar in spirit.  I would call it `+ta-tag`, `+ta-id`, `+ta-pipe`, or something like that.

Reply:
style suggestion `(pack /drum/(scot %p our.hid))` should do the same thing
(or `(pack (dray /tas/p %drum our.hid))` but that's honestly overkill)

Reply:
…who appear to no longer be around.
[These guys](https://github.com/urbit/arvo/blob/master@%7B2019-1-1%7D/sys/vane/eyre.hoon#L210-L218), though I could swear were in zuse at some point. Consider this suggestion penalized by the requirement to reintroduce library code

Reply:
gotcha, I called it a duct because of how `sole-action` is defined:
```
++  sole-action                                         ::  sole to app
  $:  id=@ta                                            ::  duct id
    $=  dat
    ...
```
but I see how `id` would be a better name